src/auth/index.ts

```diff
@@ -30,6 +30,7 @@ export class FrappeAuth {
         {
           usr: username,
           pwd: password,
+          device,
```

Comment:
Does this work if device is undefined? Or should we set a default value?

Reply:
Frappe already sets that [on their end](https://github.com/frappe/frappe/blob/a14367f1d98c05aef062ebe0f88fba1a11ccc1b4/frappe/sessions.py#L228), so this'll still work on `undefined`